tools/parse/src/test/scala/org/allenai/nlpstack/parse/poly/polyparser/GoldParseTrainingVectorSourceSpec.scala

```diff
@@ -45,6 +45,10 @@ class GoldParseTrainingVectorSourceSpec extends UnitSpec {
   )
   val parseSource = InMemoryPolytreeParseSource(List(parse1))
 
+  /** TODO: These unit tests are currently disabled because the transition systems temporarily
+    * do not permit features to be specified during construction. They will be re-enabled when
+    * these functionality returns. */
+  /*
   "Calling TrainingVectorGenerator's generate" should "create parse1's feature vectors" in {
```

Comment:
Thanks, that's good to know. Although in this case, the tests currently won't compile either.